### DISCUSSION TITLE
哪里可以下载到macOS M1 芯片的 v2ray GUI版本？

### DISCUSSION THREAD

**Discussion by JKinGH** [v2fly/v2ray-core#1447] [General]
哪里可以下载到macOS11.6  M1 芯片的 v2ray GUI版本？小白打开 导入vmess即可使用？

**Comment by kallydev** [comment]
> 哪里可以下载到macOS11.6  M1 芯片的 v2ray GUI版本？小白打开 导入vmess即可使用？

You can use [clashX](https://github.com/yichengchen/clashX/releases/tag/1.72.0), [V2rayU](https://github.com/yanue/V2rayU/releases), Surge or some iOS clients.

![image](https://user-images.githubusercontent.com/36319157/144773712-853ba53d-65c0-4b7c-8d35-5c112363a78f.jpeg)

**Comment by youkaichao** [comment]
我根据 https://www.v2fly.org/developer/intro/compile.html 这里的脚本构建部分，用 ``./user-package.sh darwin arm64 nosource nodat noconf``构建成功了，有需要的朋友可以自取：https://cloud.tsinghua.edu.cn/f/3caf8e05c7e946d4949a/

**Comment by ghost** [comment]
M1为什么不用小火箭呢？
